### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by Pluiexo**
Updated the all the images in the user guide to match the new UI. 
Updated the UI.png as well.
Added sections include, `select` ,`meeting-delete`,`Navigability` for UI controls

Fixed several typos in the UG as well(May not be comprehensive)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -183,9 +218,36 @@ Format: `meeting INDEX [d/DESCRIPTION] [s/DATETIME]`
...
+* `meeting-add 2 d/ Meet for practical exam s/ 20/04/2024 15:00` adds a meeting to the second person with the description of `Meet for practical exam` and the date and time of `20/04/2024 15:00`
+
+<br>**Result for `meeting-add 1 d/ Meet for finals preparation s/ 12/04/2024 18:00`:**
+  ![result for 'meeting-add 1 d/ Meet for finals preparation s/ 12/04/2024 18:00'](images/addMeetingResult.png)
+
+### Deleting a meeting from a person: `meeting-delete`
+
+Deletes a meeting from  person based on specified meeting index.
+
+Format: `meeting INDEX i/MEETING-INDEX `
```
Comment (whitesnowx) [inline docs/UserGuide.md]: `meeting INDEX i/MEETING-INDEX` to `meeting-delete INDEX i/MEETING-INDEX`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -310,8 +393,11 @@ Action | Format, Examples
 **Filter** | `filter [m/MODULE] [f/FACULTY] [t/TAG]… [a/AVAILABILITY]…`<br> e.g., `filter m/CS2100 t/friends`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **Sort** | `sort [ATTRIBUTE]`<br> e.g., `sort p/`
-**Add Meeting** | `meeting INDEX [d/DESCRIPTION] [s/DATETIME]`<br> e.g., `meeting 1 d/ Meet for finals preparation s/ 12/04/2024 18:00`
+**Add Meeting** | `meeting-add INDEX d/DESCRIPTION s/DATETIME`<br> e.g., `meeting-add 1 d/ Meet for finals preparation s/ 12/04/2024 18:00`
+**Delete Meeting** | `meeting-delete INDEX I/MEETING-INDEX`<br> e.g., `meeting-delete 1 i/1 `
```
Comment (whitesnowx) [inline docs/UserGuide.md]: 1. lowercase the prefix meeting index  --> `meeting-delete INDEX I/MEETING-INDEX` to
```
meeting-delete INDEX i/MEETING-INDEX 
```

2. (Just noticed) Can you help to change the Add meeting spacing between the prefix and dates (to be consistent with the rest of the format) --> `meeting-add 1 d/ Meet for finals preparation s/ 12/04/2024 18:00` 
```
meeting-add 1 d/Meet for finals preparation s/12/04/2024 18:00
```

**Review by whitesnowx — APPROVED** [review]
Looks good to me, just details to amend before merge

File: docs/UserGuide.md (hunk 2 above)
Comment (Pluiexo) [reply docs/UserGuide.md]: All fixed!

**Comment by whitesnowx** [comment]
I think the merge conflict was not done correctly?

###### Lines 337-338 for userGuide.md
The find command is removed, and sort is not up-to-date with the current one.

```
**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
**Sort** | `sort [n/] [p/] [m/] [f/] [v/] [s/] [meet/]...`<br> e.g., `sort n/ p/ m/`
```

**Comment by Pluiexo** [comment]
> I think the merge conflict was not done correctly?
> 
> ###### Lines 337-338 for userGuide.md
> The find command is removed, and sort is not up-to-date with the current one.
> 
> ```
> **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
> **Sort** | `sort [n/] [p/] [m/] [f/] [v/] [s/] [meet/]...`<br> e.g., `sort n/ p/ m/`
> ```

My bad i thought i saw no difference and accepted the wrong changes

**Comment by Pluiexo** [comment]
> > I think the merge conflict was not done correctly?
> > ###### Lines 337-338 for userGuide.md
> > The find command is removed, and sort is not up-to-date with the current one.
> > ```
> > **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
> > **Sort** | `sort [n/] [p/] [m/] [f/] [v/] [s/] [meet/]...`<br> e.g., `sort n/ p/ m/`
> > ```
> 
> My bad i thought i saw no difference and accepted the wrong changes

Fixed

**Pull request merged**